### PR TITLE
docuum: add `service` block

### PR DESCRIPTION
**Pull request by carlocab**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

See https://github.com/stepchowfun/docuum#configuring-your-operating-system-to-run-the-binary-as-a-daemon. Also, let's try to build this on Linux.

CC @stepchowfun

### CODE REVIEW

File: Formula/docuum.rb [hunk 1]
```diff
@@ -18,6 +18,16 @@ def install
     system "cargo", "install", *std_cargo_args
   end
 
+  plist_options manual: "docuum"
+  # https://github.com/stepchowfun/docuum#configuring-your-operating-system-to-run-the-binary-as-a-daemon
+  service do
+    run [opt_bin/"docuum", "--threshold", "15 GB"]
```
Comment (stepchowfun) [inline Formula/docuum.rb:24]: Just wondering, does the user have the ability to customize this threshold somehow?

**Comment by stepchowfun** [comment]
I didn't know Homebrew could manage services. This is cool!

File: Formula/docuum.rb (hunk 1 above)
Comment (carlocab) [reply Formula/docuum.rb:24]: Not that I know of, unfortunately.

Comment (stepchowfun) [reply Formula/docuum.rb:24]: In that case, perhaps it is better to not hardcode a threshold here and just rely on Docuum's default ([currently](https://github.com/stepchowfun/docuum/blob/5959ff7fab0be2fbafbfde2bed9fbdd8a7806eae/src/main.rs#L32) `10 GB`):

```ruby
run [opt_bin/"docuum"]
```

What do you think?

Comment (carlocab) [reply Formula/docuum.rb]: Works for me.

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**